pandas/tests/io/parser/test_na_values.py

```diff
@@ -138,6 +135,7 @@ def f(i, v):
     tm.assert_frame_equal(result, expected)
 
 
+# TODO: needs skiprows list support in pyarrow
```

Comment:
I think we would need to reimplement ``pyarrow_skip``/``pyarrow_xfail`` as markers instead of fixtures for that to happen.

(Currently, they are converted into markers by usefixtures).